espaloma/mm/energy.py

```diff
@@ -298,18 +306,40 @@ def energy_in_graph(
             ntype="n4_improper",
         )
 
-    if g.number_of_nodes("nonbonded") > 0 and "nonbonded" in terms:
+
+    # if g.number_of_nodes("nonbonded") > 0 and "nonbonded" in terms:
+    #     g.apply_nodes(
+    #         lambda node: apply_nonbonded(node, suffix=suffix),
+    #         ntype="nonbonded",
+    #     )
+
+    # if g.number_of_nodes("onefour") > 0 and "onefour" in terms:
+    #     g.apply_nodes(
+    #         lambda node: apply_nonbonded(
+    #             node,
+    #             suffix=suffix,
+    #             scaling=0.5,
+    #         ),
+    #         ntype="onefour",
+    #     )
+
+    if "nonbonded" in terms or "onefour" in terms:
+        esp.mm.nonbonded.get_q(g)
+
+    if g.number_of_nodes("nonbonded") > 0:
         g.apply_nodes(
-            lambda node: apply_nonbonded(node, suffix=suffix),
+            lambda node: apply_coulomb(
+                node, suffix=suffix, scaling=1.0,
+            ),
             ntype="nonbonded",
         )
 
-    if g.number_of_nodes("onefour") > 0 and "onefour" in terms:
+    if g.number_of_nodes("onefour") > 0:
         g.apply_nodes(
-            lambda node: apply_nonbonded(
-                node,
-                suffix=suffix,
-                scaling=0.5,
+            lambda node: apply_coulomb(
+                node, suffix=suffix,
+                # scaling=0.5,
+                scaling=0.8333333333333334,
```

Comment:
This is more easily expressed as `1.0/1.2`. Amber uses the `scee = 1.2` for "inverse scaling".